msteams-platform/bots/what-are-bots.md

```diff
@@ -46,17 +46,24 @@ You can use the Teams bot APIs to get information for members of a chat or team.
 <!--- TBD: For quick scanning, see if the above information can be itemized as a list.
 --->
 
-## Next step
+## Global endpoint for bots API
 
-> [!div class="nextstepaction"]
-> [Bots and SDKs](~/bots/bot-features.md)
+When creating a bot, the global routing endpoint can be enabled by setting the `cloud` parameter as `teams` in the URL. For example, the URL should be `https://smba.trafficmanager.net/teams/v3/conversations/12345`.
```

Comment:
> the global routing endpoint can be enabled by setting the `cloud` parameter as `teams` in the URL

This needs a code sample

Reply:
@aosolis what kind of example are you thinking about? I included a sample url, in the scenario the devs are calling the apis directly.  If the devs use the bot framework, i'm not sure if we want to expose this? Technically we can include something like below, but that looks more like a hack. In the BF scenario i think the framework should take care of this as an improvement if they care about. Also my knowledge of how devs are implementing bots is very limited, so i might not be thinking about all scenarios.

`      //use global routing cloud
       string globalEnpoint = "https://smba.trafficmanager.net/teams/";
       var connectorClient = turnContext.TurnState.Get<IConnectorClient>();
       connectorClient.BaseUri = new Uri(globalEnpoint); `

Reply:
This goes back to the scenario motivating this feature. You don't really need it in the reactive case, since the correct SMBA endpoint is already in the activity. But that's what this is showing.